build-tools/packages/build-cli/src/library/commands/base.ts

```diff
@@ -284,3 +289,18 @@ export abstract class BaseCommand<T extends typeof Command>
 		}
 	}
 }
+
+export abstract class BaseCommandWithBuildProject<
```

Comment:
Override the getContext function to throw an exception - hopefully will help prevent accidental uses of Context in new commands.

Reply:
You can also mark it as deprecated in this subclass - that should help prevent downstream uses too. Just extra protection.